Description: Fix dependencies (forms-rt-minimized dropped), pull the rest from
 /usr/share/java
Author: Felix Natter <fnatter@gmx.net>
Forwarded: not-needed
Last-Update: 2019-10-09
--- a/build.gradle
+++ b/build.gradle
@@ -11,9 +11,9 @@
 }
 
 repositories {
-  mavenCentral()
+  //mavenCentral()
   flatDir {
-    dirs 'libs'
+    dirs '/usr/share/java/'
   }
 }
 
@@ -35,7 +35,7 @@
 
 dependencies {
   library name: 'iconloader'
-  library name: 'forms-rt-minimized'
+  //library name: 'forms-rt-minimized'
   configurations.compile.extendsFrom configurations.library
   compile group: 'org.jetbrains', name: 'annotations', version: '16.0.1'
   testCompile group: 'junit', name: 'junit', version: '4.12'
